backend/src/main/resources/application-azure-prod.yaml

```diff
@@ -3,6 +3,7 @@ spring:
 simple-report:
   azure-reporting-queue:
     enabled: true
+    fhir-queue-enabled: false
```

Comment:
You technically don't need to overwrite this here since it's already set to `false` in the default `application.yaml`, but it doesn't hurt to keep it either. (IMO your way is a little clearer than relying on the default, just wanted to point it out)